### PR TITLE
SHOT-4351: Add Python 3.10 packages to dist

### PR DESCRIPTION
**Pull request by staceyoue**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ The main use of this framework is by the [Alias Engine](https://github.com/shotg
 
 ### <a name="support"></a>Support
 
-- Alias >= 2021
+- Alias >= 2022.0
```
[the remark this answers is not in the document]
Comment (staceyoue) [reply README.md:13]: Even better 👍

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
-[![Python 3.7 3.9](https://img.shields.io/badge/python-3.7%20%7C%203.9-blue.svg)](https://www.python.org/)
+[![VFX Platform](https://img.shields.io/badge/vfxplatform-2024%20%7C%202023%20%7C%202022%20%7C%202021-blue.svg)](http://www.vfxplatform.com/)
```
Comment (julien-lang) [inline README.md:1]: Please remove 2024. Python 3.10 is VFX 2023

**Review by julien-lang — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (staceyoue) [reply README.md]: Ah good catch, I missed that.

**Review by carlos-villavicencio-adsk — APPROVED** [review]
LGTM

**Review by julien-lang — APPROVED** [review]
LGTM!

**Pull request merged**